docs/content/release_notes/limitations.md

```diff
@@ -41,4 +45,29 @@ When a role switch is conducted, this is not reflected within the other orchestr
  2. Delete the relevant PersistentVolumes.
  3. Import the volumes, by using the latest import procedure (version 1.7.0 or later) (see **CSI driver configuration** > **Advanced configuration** > **Importing an existing volume** in the user information).
       
-    For more information, see the [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/) information in the Kubernetes documentation.
\ No newline at end of file
+    For more information, see the [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/) information in the Kubernetes documentation.
+
+## HyperSwap usage limitations
+
+**Important:** The HyperSwap feature is only supported for use with IBM Spectrum Virtualize family storage systems.
+
+The following IBM block storage CSI driver features are not supported on volumes where HyperSwap is used:
+
+- A HyperSwap volume cannot be created from a snapshot.
+
+    **Note:** A snapshot can be created from a HyperSwap volume.
+ - Volume cloning.
+
+## NVMe®/FC usage limitations
+
+ Red Hat® Enterprise Linux CoreOS (RHCOS) does not support NVMe®/FC.
+ 
+ For other limitations with your storage system, see the following section within your Spectrum Virtualize product documentation on [IBM Documentation](https://www.ibm.com/docs/en/): **Configuring** > **Host attachment** > **NVMe over Fibre Channel host attachments** > **FC-NVMe limitations and SAN configuration guidelines**.
+ 
+ ## Volume attach limitations
+
+ In cases where a volume cleanup consistently fails, eventually the orchestrator incorrectly resorts to detaching the volume. As a result, subsequent volume attachment disruptions may occur on the worker node.
+ 
+ This limitation is tracked in the following places:
+    - Red Hat Bug [2022328](https://bugzilla.redhat.com/show_bug.cgi?id=2022328)
+    - Kubernetes Issue [106710](https://github.com/kubernetes/kubernetes/issues/106710)
```

Comment:
are they supposed to [appear in a single line](https://github.com/IBM/ibm-block-csi-driver/blob/4a65c4222180b0d541ca568a27c412dd46f15e76/docs/content/release_notes/limitations.md#volume-attach-limitations)?

Reply:
done